templates/title.mustache

```diff
@@ -9,7 +9,7 @@
 					</a>
 				</li>
 				<li>
-					<a class="github-button nav-social" href="https://github.com/canjs/{{repoName src.path}}">Star</a>
+					<a class="github-button nav-social" href="https://github.com/canjs/{{repoName src.path}}" data-count-href="/canjs/canjs/stargazers" data-show-count="true">Star</a>
```

Comment:
Does `data-count-href` do anything? If so, it should be for the individual repo, not the main CanJS repo.